scss/_root.scss

```diff
@@ -137,6 +137,12 @@
   --#{$prefix}form-invalid-color: #{$form-invalid-color};
   --#{$prefix}form-invalid-border-color: #{$form-invalid-border-color};
   // scss-docs-end root-form-validation-variables
+
+  // ******* Additions for dark-mode
+  --#{$prefix}hover-color: #{$body-color};
+  --#{$prefix}active-bg: #{$gray-400};
+  --#{$prefix}disabled-color: #{$gray-500};
```

Comment:
As a first step, I'd be fine fine defining those right there, but IMHO, they should have their own Sass variables somewhere.
Fine for now but if we keep them, I like the naming though it's a bit too evasive maybe.

Reply:
Yes, variables names and way to do it are te be discussed... but for now useful and simple 😄